Extensions/Firebase/B_firebasetools/D_cloudfirestoretools.ts

```diff
@@ -329,6 +552,8 @@ namespace gdjs {
          * @param collectionName - The collection where to count documents.
          * @param [callbackValueVariable] - The variable where to store the result.
          * @param [callbackStateVariable] - The variable where to store if the operation was successful.
+         *
+         * @deprecated Use a query without filters instead.
          */
         export const listDocuments = (
```

Comment:
I think this should be deprecated as it is basically doing a query so all documents have been fetched, but it only gives the IDs of the documents to the user, making them have to refetch those individually. Using an empty query allows having access to the fetched documents data, which is a better solution.